builder/release_notes_css_prefix.txt

```diff
@@ -0,0 +1,582 @@
+    *width: 13px; /* 3 */
+}
+
+/**
+ * 1. Address `appearance` set to `searchfield` in Safari 5 and Chrome.
+ * 2. Address `box-sizing` set to `border-box` in Safari 5 and Chrome
+ *    (include `-moz` to future-proof).
+ */
+
+input[type="search"] {
+    -webkit-appearance: textfield; /* 1 */
+    -moz-box-sizing: content-box;
+    -webkit-box-sizing: content-box; /* 2 */
+    box-sizing: content-box;
+}
+
+/**
+ * Remove inner padding and search cancel button in Safari 5 and Chrome
+ * on OS X.
+ */
+
+input[type="search"]::-webkit-search-cancel-button,
+input[type="search"]::-webkit-search-decoration {
+    -webkit-appearance: none;
+}
+
+/**
+ * Remove inner padding and border in Firefox 3+.
+ */
+
+button::-moz-focus-inner,
+input::-moz-focus-inner {
+    border: 0;
+    padding: 0;
+}
+
+/**
+ * 1. Remove default vertical scrollbar in IE 6/7/8/9.
+ * 2. Improve readability and alignment in all browsers.
+ */
+
+textarea {
+    overflow: auto; /* 1 */
+    vertical-align: top; /* 2 */
+}
+
+/* ==========================================================================
+   Tables
+   ========================================================================== */
+
+/**
+ * Remove most spacing between table cells.
+ */
+
+
+</style>
+
+<style type="text/css">
+
+/*
+Theme Name: Livecode 2013
+Theme URI: http://www.livecode.org/
+Description: A custom theme for LiveCode
+Version: 1.0
+Author: James Cant & Benjamin Beaumont
+Author URI: http://www.livecode.com/
+*/
+
+/***************************************************************************
+ * FONTS
+****************************************************************************/
+@font-face{ font-family: MuseoSansForDellBold; font-style: bold; font-weight: bold; src: url(http://livecode.com/wp-content/themes/livecode2013/fonts/museosansfordell-bold-webfont.eot), url(http://livecode.com/wp-content/themes/livecode2013/fonts/museosansfordell-bold-webfont.eot?#iefix) format("embedded-opentype"), url(fonts/museosansfordell-bold-webfont.woff) format("woff"), url(http://livecode.com/wp-content/themes/livecode2013/fonts/museosansfordell-bold-webfont.ttf) format("truetype"), url(http://livecode.com/wp-content/themes/livecode2013/fonts/museosansfordell-bold-webfont.svg#MuseoSansForDellBold) format("svg"); }
+
+@font-face{ font-family: MuseoSansForDell; font-style: normal; font-weight: normal; src: url(http://livecode.com/wp-content/themes/livecode2013/fonts/museosansfordell-regular-webfont.eot), url(http://livecode.com/wp-content/themes/livecode2013/fonts/museosansfordell-regular-webfont.eot?#iefix) format("embedded-opentype"), url(http://livecode.com/wp-content/themes/livecode2013/fonts/museosansfordell-regular-webfont.woff) format("woff"), url(http://livecode.com/wp-content/themes/livecode2013/fonts/museosansfordell-regular-webfont.ttf) format("truetype"), url(http://livecode.com/wp-content/themes/livecode2013/fonts/museosansfordell-regular-webfont.svg#MuseoSansForDellRegular) format("svg"); }
+
```

Comment:
Could you update the `@font-face` to the lone
`@font-face{ font-family: FontAwesome; font-style: normal; font-weight: normal; src: url(http://livecode.com/wp-content/themes/omega/assets/fonts/fontawesome-webfont.eot) format("embedded-opentype"), url(http://livecode.com/wp-content/themes/omega/assets/fonts/fontawesome-webfont.woff) format("woff"), url(http://livecode.com/wp-content/themes/omega/assets/fonts/fontawesome-webfont.ttf) format("truetype"), url(http://livecode.com/wp-content/themes/omega/assets/fonts/fontawesome-webfont.svg) format("svg"); }`